styles/app.css

```diff
@@ -1,3 +1,30 @@
 :focus:not(:focus-visible) {
   outline: none;
 }
+
+input:-webkit-autofill,
+input:-webkit-autofill:hover,
+input:-webkit-autofill:focus,
+input:-webkit-autofill:active {
+  -webkit-text-fill-color: black !important;
+  -webkit-box-shadow: 0 0 0 999px #e6e9ee inset !important; /* gray-200 */
```

Comment:
Maybe we should create a css variable so this and `gray-200` can use the same thing (also allows us to change it for light/dark mode as needed).

Reply:
Somewhat done with ae287d39f8f2940264167f65bf79cadf7cf26ca6. I think we can build upon that to improve things futher. Think of slimming down the gray-pallete, and assign colors like. I need to research a bit, but I think we can reduce our current 8 gray colors, to 4 (per theme) by doing so.

```jsx
<div className="bg-gray-dark" />
```

instead of:

```jsx
<div className="bg-gray-200 dark:bg-gray-800" />
```